scenes/quests/lore_quests/quest_001/1_music_puzzle/components/bonfire_sign/bonfire_sign.tscn

```diff
@@ -186,4 +187,15 @@ action = "Listen"
 position = Vector2(2, 0)
 shape = SubResource("CircleShape2D_gye5d")
 
+[node name="FireSignSound" type="AudioStreamPlayer2D" parent="."]
+stream = ExtResource("13_rd7vt")
+volume_db = -3.0
+max_distance = 300.0
+parameters/looping = true
+
+[node name="FireStart" type="AudioStreamPlayer2D" parent="."]
+stream = ExtResource("13_6s45h")
+volume_db = -1.0
+max_distance = 300.0
```

Comment:
I wonder if the distance wants to be a little bit larger – I think I couldn't hear one of the fires start because I was at a rock far away from it.